source/FreeRTOS_Routing.c

```diff
@@ -241,10 +241,6 @@ struct xIPv6_Couple
     {
         NetworkEndPoint_t * pxIterator = NULL;
 
-        /* This end point will go to the end of the list, so there is no pxNext
```

Comment:
Yeah i think this needs to be removed in the case the endpoint is already present on the list.